catalogs/catalog-model/build.gradle.kts

```diff
@@ -40,11 +40,17 @@ dependencies {
     exclude(group = "*")
   }
 
-  compileOnly(libs.guava)
-
+  implementation(libs.guava)
   implementation(libs.slf4j.api)
 
+  testImplementation(project(":clients:client-java"))
+  testImplementation(project(":integration-test-common", "testArtifacts"))
```

Comment:
Why is this necessary?

Reply:
It is not used currently, will be used when integration test is added.